### PR TITLE
Update changelog and known issues for July 26 prod push

### PR DESCRIPTION
**Pull request by bheftel**
- Update changelog for the recent bug fixes in the beta UI, including assorted cosmetic issues and the slack integration bug
- Update the status of the slack integration known issue. 
    - @richbrowne and @Jcahilltorre - we might want to change the format of the known issues page to match NGINXaaS for Azure - https://docs.nginx.com/nginxaas/azure/known-issues/ - they aren't showing a status, they are just removing issues once they are fixed

### CODE REVIEW

File: content/changelog.md [hunk 1]
```diff
@@ -5,6 +5,16 @@ weight: 800
 toc: true
 ---
 
+## July 26, 2023
+
+### What's New
+ 
+- {{% icon-resolved %}} **Configure Slack Notifications in Beta UI and other bug fixes**
```
Comment (Jcahilltorre) [inline content/changelog.md:12]: this should use a different Icon

Comment (bheftel) [reply content/changelog.md:12]: Which icon should it use? I was looking at https://docs.nginx.com/nginxaas/azure/changelog/ as an example

Comment (Jcahilltorre) [inline content/changelog.md:12]:
```suggestion
- {{% icon-feature %}} **Configure Slack Notifications in Beta UI and other bug fixes**
```

Comment (bheftel) [reply content/changelog.md:12]: thanks, I'll make the update.

Is there guidance on when the bug resolved icon should be used?

**Pull request merged**